app/components/ui/Navigation/Navigation.jsx

```diff
@@ -46,27 +57,23 @@ class Navigation extends React.Component {
     // sites, we want to just use an internal react-router Link to the root URL,
     // since 1) this allows us to use react-router routing and 2) this avoids
     // having staging and development environments link to the production site.
-    let logoLink;
-    if (isSFFamiliesSite()) {
-      logoLink = (
-        <a className={styles.navLogoSFFamilies} href={getSiteUrl()}>
-          <img src={images.logoSmall} alt={getSiteTitle()} />
-        </a>
-      );
-    } else {
-      logoLink = (
-        <Link className={styles.navLogo} to="/">
-          <img src={images.logoSmall} alt={getSiteTitle()} />
-        </Link>
-      );
-    }
-
     return (
-      <nav className={isSFFamiliesSite() ? styles.siteNavSFFamilies : styles.siteNav}>
+      <nav className={siteNavStyle}>
         <div className={styles.primaryRow}>
-          <div className={styles.navLeft}>
-            {logoLink}
-            {showSearch
+          {
+            /^https?:\/\//.test(logoLinkDestination)
+              ? (
+                <a className={navLogoStyle} href={siteUrl}>
+                  <img src={logoSmall} alt={title} />
+                </a>
+              )
+              : (
+                <Link className={navLogoStyle} to="/">
```

Comment:
For consistency, you should also set the `to` prop here to `logoLinkDestination`, which would better mirror what's going on in the other case.

Reply:
Acknowledged. Good Ideas, I agree with all of them. Implementing.